pennylane/qaoa/mixers.py

```diff
@@ -23,35 +23,22 @@
 
 def x_mixer(wires):
     r"""Creates a basic Pauli-X mixer Hamiltonian.
-
     This Hamiltonian is defined as:
-
     .. math:: H_M \ = \ \displaystyle\sum_{i} X_{i},
-
     where :math:`i` ranges over all wires, and :math:`X_i`
     denotes the Pauli-X operator on the :math:`i`-th wire.
-
     This is mixer is used in *A Quantum Approximate Optimization Algorithm*
     by Edward Farhi, Jeffrey Goldstone, Sam Gutmann [`arXiv:1411.4028 <https://arxiv.org/abs/1411.4028>`__].
-
-
     Args:
         wires (Iterable or Wires): The wires on which the Hamiltonian is applied
-
     Returns:
         Hamiltonian: Mixer Hamiltonian
-
     .. UsageDetails::
-
         The mixer Hamiltonian can be called as follows:
-
         .. code-block:: python
-
             from pennylane import qaoa
-
             wires = range(3)
             mixer_h = qaoa.x_mixer(wires)
-
         >>> print(mixer_h)
```

Comment:
These line deletions don't look right

Reply:
@Lucaman99, these line deletions were the reason the documentation couldn't build. I've reverted it, and now it works